src/controllers/bookings.ts

```diff
@@ -1,11 +1,20 @@
 import { Response, Request, NextFunction } from 'express'
 import { HttpCode, HttpException } from '@exceptions/HttpException'
+import { Booking } from '@prisma/client'
+import { addBooking } from '@services/bookings'
 
 export async function createBooking(
   req: Request,
-  res: Response
+  res: Response,
+  next: NextFunction
 ): Promise<void> {
-  res.send('create booking!')
+  try {
+    const booking = JSON.parse(req.body) as Booking
```

Comment:
shld this be using `BookingSchema.parse(...)` ?

Reply:
sorry i only updated this in my other PR, but yup it should be that!